### PR TITLE
DOCS-2042: Add board notice

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/components/board/_index.md [hunk 1]
```diff
@@ -33,6 +33,16 @@ Signaling is overseen by a computer running `viam-server` which allows you to co
 
 To use your board with Viam, check whether one of the following [built-in models](#built-in-models) or [modular resources](#modular-resources) supports your board.
 
+{{< alert title="Running viam-server" color="note" >}}
+
+The board component allows you to use the pins on your board.
+If there is no board model for your board:
+
+- you can still run `viam-server` if your board [supports it](/get-started/installation/#prepare-your-board)
```
Comment (EshaMaharishi) [inline docs/components/board/_index.md:41]: looks like a typo - should this link to /get-started/installation/#compatibility?  :)

**Review by EshaMaharishi — APPROVED** [review]
Thanks for jumping on this so quickly! I may have a suggestion for condensing/rewording the first few paragraphs on the page a bit to also make the distinction between "run on board" and "board is collection of GPIO pins" more clear, but want to chat with Rand to make sure what I suggest makes sense first, so going ahead and approving this and if you like I an submit a separate PR for any additional updates?

File: docs/components/board/_index.md (hunk 1 above)
Comment (npentrel) [reply docs/components/board/_index.md:41]: yes - thanks!

Comment (npentrel) [inline docs/components/board/_index.md:41]:
```suggestion
- you can still run `viam-server` if your board [supports it](/get-started/installation/#compatibility)
```

**Comment by npentrel** [comment]
That would be great - thanks @EshaMaharishi !

**Pull request merged**